internal/policy/source/source.go

```diff
@@ -170,6 +189,45 @@ func (p *PolicyUrl) Subdir() string {
 	return string(p.Kind)
 }
 
+func (p PolicyUrl) Type() PolicyType {
+	return p.Kind
+}
+
+func getPinnedUrl(u string, m metadata.Metadata) (string, error) {
```

Comment:
I hope that eventually this moves to go-gather itself. Could you add a TODO comment if you agree?

Reply:
Added.